sdks/python/apache_beam/dataframe/expressions.py

```diff
@@ -24,60 +30,77 @@ class Session(object):
   def __init__(self, bindings={}):
     self._bindings = dict(bindings)
 
-  def evaluate(self, expr):
+  def evaluate(self, expr):  # type: (Session, Expression) -> Any
```

Comment:
Here (and throughout) I don't think the convention is to declare the type of the self parameter for methods.

Reply:
Done